scripts/generate_screenshots/apply_variant.sh

```
@@ -0,0 +1,75 @@
+for lang_dir in "$VARIANT_DIR"/*/; do
+  if [ ! -d "$lang_dir" ]; then
+    continue
+  fi
+
+  lang=$(basename "$lang_dir")
+  dest_dir="$FASTLANE_DIR/$lang"
+  mkdir -p "$dest_dir"
+
+  # バリアントのスクリーンショットを適用先にコピー（上書き）
+  cp -f "$lang_dir"*.png "$dest_dir/" 2>/dev/null || true
+  file_count=$(ls -1 "$lang_dir"*.png 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)